README.md

```diff
@@ -0,0 +1,53 @@
+Quattor Dashboard
+=================
+
+The Quattor Dashboard provides a web based interface to Quattor.
+Information are gathered by scripts located on the AII server and a SVN client host :
+
+* AII server
+    - Must have access to JSON profiles and the `aii-shellfe` command
+* SVN client
+    - Must be able to call `svn log --xml $repo`
+
+Software
+========
+
+* CherryPy 3.1.1 + Jinja2
+* Apache 2.2.15 + mod_wsgi + mod_proxy
+* Bootstrap 3.1.1
+* jQuery 1.11.0 + DataTables 1.10.5 + jqPlots 1.0.8
+* CollapsibleLists
+* lessc 1.7.0 or greater (for css modifications)
+
+Deployment
+==========
+
+1- Install required software & scripts
+
+* On the web server
+    - `yum install httpd mod_wsgi mod_proxy git python-cherrypy python-jinja2`
+    - `git clone https://github.com/IIHE/dashboard.git`
+    - Copy the vhost-dashboard.conf to your Apache config directory (eg. /etc/http/conf.d/) and replace the following:
+      * AQ_URL - URL of your Aquilon server (including port)
+      * AII_URL - URL of your AII server
+      * SVN_URL - URL of your SVN server
+      * DASHBOARD_DIR - The absolute path of this directory
+      * MY_HOST - The hostname of the webserver
+    - If you want to hide aquilon related menu : Edit assets/js/pages/config.js and set AQ_URL to ''.
+
+* On the AII server
+    - `yum install perl-Config-Tiny`
```

Comment:
but if this an AII-server, it will already be there since it's a dependency of CAF